backend/models/office_hours/oh_section_details.py

```diff
@@ -0,0 +1,20 @@
+from ..academics.section import Section
+from oh_event import OfficeHoursEvent
+from oh_section import OfficeHoursSection
+
+__authors__ = ["Sadie Amato, Bailey DeSouza, Meghan Sun, Maddy Andrews"]
+__copyright__ = "Copyright 2024"
+__license__ = "MIT"
+
+
+class OfficeHoursSectionDetails(OfficeHoursSection):
+    """
+    Pydantic model to represent an `OfficeHoursSection`, including back-populated
+    relationship fields.
+
+    This model is based on the `OfficeHoursSectionEntity` model, which defines the shape
+    of the `OfficeHoursSection` database in the PostgreSQL database.
+    """
+
+    sections: list[Section]
+    events: list[OfficeHoursEvent]
```

Comment:
It seems like there may need to be another model for getting upcoming events to drive an office hours section home page as opposed to all events in the history of a section.